### PR TITLE
AF-27:AppFrameworkService getAllExtensions() method should return all…

### PR DESCRIPTION
**Pull request by gitcliff**
LINK:https://issues.openmrs.org/browse/AF-27
AppFrameworkService getAllExtensions() method should return all extensions

### CODE REVIEW

**Comment by gitcliff** [comment]
@mogoodrich @dkayiwa  requesting for your review

File: api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java [hunk 1]
```diff
@@ -154,6 +154,13 @@ public void testGetAllExtensionsAndIsSortedByOrder() throws Exception {
 
     }
 
+	@Test
+	public void testGetAllExtensionsWithNoExtensionPointId() throws Exception {
```
Comment (HerbertYiga) [inline api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java]: you inroduced getAllExtensions() but i am seeing the name of the method you are testing is different!!

Comment (gitcliff) [reply api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java]: its the same

**Comment by mogoodrich** [comment]
Unfortunately I think we need to close this out without merging... I don't remember 100% the intent of this ticket, and it's 7 years old, I just closed it as won't fix.

**Pull request closed without merging**